payplug/test/test_resources/test_accounting_report.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+from mock import patch
+import payplug
+from payplug.resources import AccountingReport
+from payplug.test import TestBase
+
+
+@patch('payplug.config.secret_key', 'a_secret_key')
+class TestAccountingReportResource(TestBase):
+    def test_initialize_accounting_report(self):
+        report_attributes = {
+            'start_date': '2020-01-01',
+            'object': 'accounting_report',
+            'notification_url': 'notification_url',
+            'end_date': '2020-04-30',
+            'id': 'ar_1GKEACvltTVXT5muBd3AQv',
+            'file_available_until': 1588083743,
+            'temporary_url': 'temporary_url'
```

Comment:
Je sais que c'est juste un mock, mais pour être exhaustif : on avait oublié un champ `is_live`, booléen, dans la resource, pour déclarer si la resource est une resource de sandbox ou de live (ce champ existe dans toutes les autres resources).

On a ajouté ce champ hier, tu pourras mettre à jour le test pour que ce soit raccord ?

Reply:
Le lien vers le preprint de la nouvelle doc si tu veux : https://improvements-ref-payplug-api-doc.payplug.rocks/apiref.html#the-accounting-report-object

Reply:
C'est rajouté !